blazor/predefined-dialogs/customization.md

```diff
@@ -39,16 +39,26 @@ For prompt dialog , customized the default dialog buttons content as `Connect` a
 
 {% endtabs %}
 
+## Output images
+
```

Comment:
Don't use the heading for this.

Reply:
Hi @thangavele 

Then how to use sir.

Reply:
**Results from the code snippet**

**Alert**

[image]

**Confirm**
[image]

**Prompt**
[image]

Use the bold instead of heading @Kokila13081999

Reply:
Hi @thangavele 

Review comments addressed